docs/ref/contrib/auth.txt

```diff
@@ -446,6 +446,13 @@ The following backends are available in :mod:`django.contrib.auth.backends`:
     implement them other than returning an empty set of permissions if
     ``obj is not None``.
 
+    .. attribute:: ModelBackend.allow_inactive_users
+
+        .. versionadded:: 1.10
+        
```

Comment:
remove whitespace on blank line (trying configuring your editor to remove trailing whitespace on save)